### PR TITLE
Adding sQTL derived features to l2g scoring

### PR DESCRIPTION
**Pull request by DSuveges**
The modifications were introduced by @Jeremy37 , and the upcoming data is generated by this code.

### CODE REVIEW

File: 1_feature_engineering/1_prepare_inputs.py [hunk 1]
```diff
@@ -23,16 +23,16 @@ def main():
     out_gs = 'gs://genetics-portal-dev-staging/l2g/{v}/features/inputs/'.format(v=args.version)
     manifest = {
         'study': {
-            'in': 'gs://genetics-portal-dev-staging/v2d/220208/studies.parquet',
+            'in': 'gs://genetics-portal-dev-staging/v2d/220210/studies.parquet',
```
Comment (ireneisdoomed) [inline 1_feature_engineering/1_prepare_inputs.py]: This study table is not up to date with the one being used in the rest of the pipeline: `gs://genetics-portal-dev-staging/v2d/220401/studies.parquet`

File: 1_feature_engineering/1_prepare_inputs.py [hunk 2]
```diff
@@ -23,16 +23,16 @@ def main():
...
             'out': out_gs + 'studies.parquet'
         },
         'toploci': {
-            'in': 'gs://genetics-portal-dev-staging/v2d/220208/toploci.parquet',
+            'in': 'gs://genetics-portal-dev-staging/v2d/220210/toploci.parquet',
```
Comment (ireneisdoomed) [inline 1_feature_engineering/1_prepare_inputs.py]: The same goes for the top loci dataset: `gs://genetics-portal-dev-staging/v2d/220401/toploci.parquet`

File: 1_feature_engineering/1_prepare_inputs.py [hunk 3]
```diff
@@ -23,16 +23,16 @@ def main():
...
             'out': out_gs + 'toploci.parquet'
         },
         'credsets_v2d': {
             'in_fm': 'gs://genetics-portal-dev-staging/finemapping/220113_merged/credset/part-*.json.gz',
-            'in_pics': 'gs://genetics-portal-dev-staging/v2d/220208/ld.parquet',
+            'in_pics': 'gs://genetics-portal-dev-staging/v2d/220210/ld.parquet',
```
Comment (ireneisdoomed) [inline 1_feature_engineering/1_prepare_inputs.py]: ... And the LD matrix: `gs://genetics-portal-dev-staging/v2d/220401/ld.parquet/`

File: README.md [hunk 4]
```diff
@@ -27,10 +30,7 @@ conda env create -n l2g --file environment.yaml
 conda activate l2g
 export PYSPARK_SUBMIT_ARGS="--driver-memory 100g pyspark-shell"
 version_date=`date +%y%m%d`
-#version_date=211108
-#version_date=220113
-#version_date=220128
-version_date=220208
+version_date=220212
```
Comment (ireneisdoomed) [inline README.md:33]: The latest version is `220712`

**Review by ireneisdoomed — COMMENTED** [review]
The code to extract the features from the sQTLs is fine. However we need to review the data that has been used to generate the data of the latest L2G run (`gs://genetics-portal-dev-staging/l2g/220712/`)

Basically the inputs that this config describe are not the most up to date. @xyg123, could you please tell us if that is the data that has been used?

**Comment by xyg123** [comment]
Morning Irene,  yes, these were not the files used for the latest (220712), the config file I used has not been pushed to this branch.  But the input files I used were the correct ones.

**Comment by ireneisdoomed** [comment]
> Morning Irene, yes, these were not the files used for the latest (220712), the config file I used has not been pushed to this branch. But the input files I used were the correct ones.

Great, thanks! Could you please update the config with your used inputs? You can open a PR to this branch or commit to this branch directly. Thanks!

File: 1_feature_engineering/1_prepare_inputs.py [hunk 5]
```diff
@@ -23,34 +23,34 @@ def main():
...
             'out': out_gs + 'credsets_qtl.parquet'
         },
         'coloc': {
-            'in': 'gs://genetics-portal-dev-staging/coloc/220127/coloc_processed_w_betas_fixed.parquet',
+            'in': 'gs://genetics-portal-dev-staging/coloc/220408/coloc_processed_w_betas.parquet',
             'out': out_gs + 'coloc.parquet'
         },
         'v2g': {
-            'in': 'gs://genetics-portal-dev-data/22.01.2/outputs/v2g/part-*.json',
+            'in': 'gs://genetics-portal-dev-data/22.05.2/outputs/parquet/v2g/part-*.json',
```
Comment (ireneisdoomed) [inline 1_feature_engineering/1_prepare_inputs.py]:
```suggestion
            'in': 'gs://genetics-portal-dev-data/22.05.2/outputs/json/v2g/part-*.json',
```

The pipeline right now expects JSON files. In the future we will be only working with parquet.

**Pull request merged**